log/loggerInterface.go

```diff
@@ -22,6 +23,23 @@ type Logger interface {
 	Infof(format string, args ...interface{})
 }
 
+// Entry describes a log entry
+type Entry struct {
```

Comment:
`Entry` might make the library to similar to logrus. Maybe `LogData`?

Reply:
`LogEntry`? The entry already contains a `Data` object.